covid_moonshot/analysis/plots.py

```diff
@@ -186,52 +188,60 @@ def plot_convergence(
 
     fig, (ax1, ax2) = plt.subplots(nrows=2, sharex=True)
 
-    DDG = np.array(solvent_delta_fs) - np.array(complex_delta_fs)
-    DDG_kcal = (DDG * KT).value_in_unit(unit.kilocalories_per_mole)
+    complex_delta_fs_kcal = pd.Series(
+        (np.array(complex_delta_fs) * KT).value_in_unit(unit.kilocalories_per_mole),
```

Comment:
There's a slightly simpler idiom than `x.value_in_unit`
```python
        np.array(complex_delta_fs) * KT / unit.kilocalories_per_mole,
```

Reply:
Cool, love this!

Reply:
Simplified in https://github.com/choderalab/covid-moonshot-infra/pull/45/commits/b2225b183ebe34892abe461b066ab977ffb1531d